rust/src/http2/http2.rs

```diff
@@ -167,6 +172,27 @@ impl HTTP2Transaction {
         if let Some(state) = self.de_state {
             core::sc_detect_engine_state_free(state);
         }
+        if self.file_range != std::ptr::null_mut() {
+            match unsafe { SC } {
+                None => panic!("BUG no suricata_config"),
+                Some(c) => {
+                    //TODO get a file container instead of NULL
```

Comment:
TODO: address or remove comment if its not needed

Reply:
It will be much easier when we have file per tx 

Right now, I should do something like `state.files.get(STREAM_TOCLIENT)` but I do not have an easy reference to `state` in `HTTP2Transaction::free`

So, what should we do ?